front-end/virtual_pet/lib/pages/home.dart

```diff
@@ -19,77 +19,79 @@ class HomeState extends State<Home> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-        backgroundColor: Theme.of(context).canvasColor,
-        body: const Center(
-          child: VirtualPetGameLayer(),
-        ),
+      backgroundColor: Theme.of(context).canvasColor,
```

Comment:
Hey @wu-jacob , good job on figuring out how to change the background :) Since the background colour that we see is coming from the `VirtualPetGameLayer` widget, this `backgroundColor` property is redundant because it is behind the `VirtualPetGameLayer` widget. So, we should remove this one here since it isn't needed

Reply:
@lou-weng Yes, because it is painted over the background of parent container. Shouldn't the background of the game layer be set in some fashion then? Why were you thinking that we shouldn't be changing the background in the game layer?